src/Features/Lsif/Generator/Generator.cs

```diff
@@ -130,7 +130,10 @@ public static Generator CreateAndWriteCapabilitiesVertex(ILsifJsonWriter lsifJso
                 }
             };
 
-            var documents = (await project.GetAllRegularAndSourceGeneratedDocumentsAsync(cancellationToken)).ToList();
+            var documents = new List<Document>();
```

Comment:
>var documents = new List<Document>(); [](http://example.com/codeflow?start=12&length=37)

Was something wrong here? Was the loop below intended to be changed to use the new parallel methods?

Reply:
no.  i just needed to update this code because GetAllRegularAndSourceGeneratedDocumentsAsync now returns an IAsyncEnumerable.